charts/nutanix-csi-storage/templates/ntnx-csi-controller-deployment.yaml

```diff
@@ -96,7 +114,7 @@ spec:
           imagePullPolicy: {{ .Values.imagePullPolicy }}
           args:
           - --csi-address=$(ADDRESS)
-          - --leader-election=false
+          - --leader-election=true
```

Comment:
Add resources for csi-snapshotter like in master branch
          resources:
            requests:
              cpu: 5m
              memory: 30Mi